### ISSUE TITLE
Add. 17,217, foll. 20–32 | https://bl.vuexistapps.us/manuscript/931 (Part2)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#994]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- I encountered an article addressing a part of this manuscript (foll. 20-32). The author has made a few revisions regarding the physical description of the manuscript. In Wright's catalog, foll. 20-32 are said to contain Theodore of Mopsuestia's commentary on Genesis. Rambow, however, observes that fol. 32 does not include commentary on Genesis, but another work of Theodore, that is, his commentary on the Minor Prophets. In other words, the manuscript of Commentary on Genesis consists of 12 leaves (Wright has 13) from fol. 23 to fol. 31. 

The bibliographic reference to this article is as follows:

Rambow, George F. “Theodore vs. the ‘Arians’ and the Parable of Humanity’s Creation: A Syriac Fragment of Theodore of Mopsuestia’s Commentary on Genesis 1.26.” Journal of Early Christian Studies 25:2 (2017): 231–54.

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Question:
Since we are encoding Wright's work, I assume we can disregard the corrections in the article?

@wlpotter

**Comment by wlpotter** [comment]
Yes, for encoding Wright we wouldn't want to make these corrections. But let's keep this info (I'll leave this issue open and add a label), since we'll want to eventually incorporate this info
